src/rust/cryptography-x509-validation/src/policy/mod.rs

```diff
@@ -107,6 +109,38 @@ pub enum Subject<'a> {
     IP(IPAddress),
 }
 
+impl Subject<'_> {
+    fn general_name_matches(&self, general_name: &GeneralName<'_>) -> bool {
```

Comment:
Can we name this `subject_alt_name_matches` or something like that -- I want to distinguish between the name constraints matching use case, which is subtly different.

Reply:
[8ecaafb](https://github.com/pyca/cryptography/pull/9659/commits/8ecaafbc6d1332c3afce7ce3939faa87a0ab4af7)